cli/src/main/package/functions

```diff
@@ -93,7 +93,9 @@ function icd() {
 _ide_completion()
 {
   if [ -z "${COMP_WORDS[COMP_CWORD]}" ]; then
-    COMPREPLY=( $(ideasy -q complete ${COMP_WORDS[@]:1} "") )
+    set -f
+    COMPREPLY=( $(ideasy -q complete ${COMP_WORDS[@]:1} "" ) )
+    set +f
   else
     COMPREPLY=( $(ideasy -q complete ${COMP_WORDS[@]:1}) )
```

Comment:
This is a great finding and fix.
However, my thoughts on this are:
* shouldn't we do `set -/+f` for all completions - so also for the one in `else` or to be more explicitly around the entire `if` block?
* with `set -f` we may tweak the shell of the user. Therefore it is great that you try to "undo" this effect afterwards. Just thinking out loud: What would happen if the user actually does `set -f` in his terminal because he for some reason does not want to have filename expansion? Then here `set -f` would have no effect but `set +f` would not reset the previous state.

Reply:
For the second one I tried some things. It git-bash you can do this:
```
$ function foo() { local -; set -f; echo *; }

$ foo
*

$ echo *
backups conf scripts settings software system updates urls workspaces
```

However, `local -` is bash specific and in `zsh` we would need to do `set -o localoptions` instead.
Even worse `local -` is only supported by bash >= 4.4 and on MacOS this is not given so it would then totally fail.
As the second point seems more of a nice-to-have to support edge-cases, I would say,  that you can ignore it in this PR and maybe you could create a new story for this that we could put in the `Research` state/column on our board.